usaon_vta_survey/util/db/user.py

```diff
@@ -8,9 +8,9 @@ def ensure_user_exists(google_json: dict) -> User:
     Note: Does this include user updates for something like a name change?
     """
     user = User(id=google_json['email'], name=google_json['name'])
-    if user := db.session.get(User, user.id):
```

Comment:
This old version actually looks better than the new version.

The old one will look up a user and return it if found, otherwise it will create it.

The new version will look up the user in the db, and if it doesn't exist, it will create and return it. If the user _does_ exist in the db, it will throw that user away and return a partially-initialized user instead of the one in the db.

I think we can just go back to the way it was here, unless I'm missing something.

Reply:
Let's look deeper at that error.